genienlp/tasks/almond.py

```diff
@@ -476,7 +478,6 @@ def get_splits(self, root, **kwargs):
 class AlmondMultiLingual(BaseAlmondMultiLingualTask):
     def __init__(self, name, args):
         super().__init__(name, args)
-        self.lang_as_question = args.almond_lang_as_question
```

Comment:
I remember this was helpful in multilingual setting when mixing English and foreign. @Mehrad0711 ?

Reply:
@Mehrad0711 said this didn't help. Some of the multilingual models we use have other means to detect languages (like a language-specific BOS token) or are trained to implicitly detect the language.
But I'm reverting this change anyways, to keep the multi-task learning option open.

Reply:
Yeah, it's not really needed. Only mbart needs language specific id which is handled in this PR #70